packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -611,6 +610,9 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
             this.channelDeferred.resolve(this.channel);
         } catch (error) {
             this.channelDeferred?.reject(error);
+            this.logger.sendErrorEvent(
+                { eventName: "bindRuntimeError", fluidDataStoreId: this.id },
```

Comment:
Same comment as above.  Also, we should be consistent about casing for event names, at least within the same PR ;)  They seem to mostly be CamelCase, so probably stick with that?